mpas_analysis/ocean/climatology_map_argo.py

```diff
@@ -0,0 +1,461 @@
+        # mpasClimatologyTask along with the seasons.
+        remapClimatologySubtask = RemapDepthSlicesSubtask(
+            mpasClimatologyTask=mpasClimatologyTask,
+            parentTask=self,
+            climatologyName=fieldName,
+            variableList=[mpasFieldName],
+            seasons=seasons,
+            depths=depths,
+            comparisonGridNames=comparisonGridNames,
+            iselValues=iselValues)
+
+        if refConfig is None:
+
+            refTitleLabel = 'Roemmich-Gilson Argo Climatology: Temperature'
+
+            observationsDirectory = build_config_full_path(
+                config, 'oceanObservations', 'argoSubdirectory')
+
+            obsFileName = \
+                '{}/ArgoClimatology_TS.nc'.format(
+                        observationsDirectory)
+            refFieldName = 'thetaArgo'
+            outFileLabel = 'tempArgo'
+            galleryName = 'Roemmich-Gilson Climatology: ARGO'
+            diffTitleLabel = 'Model - Argo'
+
+            remapObservationsSubtask = RemapArgoClimatology(
+                    parentTask=self, seasons=seasons, fileName=obsFileName,
+                    outFilePrefix=refFieldName,
+                    fieldName=refFieldName,
+                    depths=depths,
+                    comparisonGridNames=comparisonGridNames)
+
+            self.add_subtask(remapObservationsSubtask)
+
+        else:
+            remapObservationsSubtask = None
+            refRunName = refConfig.get('runs', 'mainRunName')
+            galleryName = 'Ref: {}'.format(refRunName)
+            refTitleLabel = galleryName
+
+            refFieldName = mpasFieldName
+            outFileLabel = 'temp'
+            diffTitleLabel = 'Main - Reference'
+
+        for comparisonGridName in comparisonGridNames:
+            for season in seasons:
+                for depth in depths:
+                    subtask = PlotClimatologyMapSubtask(
+                        parentTask=self,
+                        season=season,
+                        comparisonGridName=comparisonGridName,
+                        remapMpasClimatologySubtask=remapClimatologySubtask,
+                        remapObsClimatologySubtask=remapObservationsSubtask,
+                        refConfig=refConfig,
+                        depth=depth)
+
+                    subtask.set_plot_info(
+                        outFileLabel=outFileLabel,
+                        fieldNameInTitle='Temperature',
+                        mpasFieldName=mpasFieldName,
+                        refFieldName=refFieldName,
+                        refTitleLabel=refTitleLabel,
+                        diffTitleLabel=diffTitleLabel,
+                        unitsLabel=r'$^\circ$C',
+                        imageCaption='Model temperature compared with ARGO observations',
+                        galleryGroup='Argo Temperature',
+                        groupSubtitle=None,
+                        groupLink='temp',
```

Comment:
@vanroekel, one more small thing.  This link name should be different for SOSE and ARGO.  I clicked on the image for ARGO in my test and it went to SOSE instead.  Sorry I didn't notice this before.  (Same for salinity, of course).  Please add 'Argo' and the end here and 'Sose' in the corresponding place in the SOSE tasks.